server/activity/activity.model.js

```diff
@@ -75,26 +75,28 @@ class Activity extends Model {
   }
 
   static async create(data, opts) {
-    return this.sequelize.transaction(async transaction => {
-      const activity = await super.create(data, { ...opts, transaction });
-      if (activity.isTrackedInWorkflow) {
-        const defaultStatus = getDefaultActivityStatus(activity.type);
-        await activity.createStatus(defaultStatus, { transaction });
-      }
-      return activity;
-    }, { transaction: opts.transaction });
+    return this.sequelize.transaction({ transaction: opts.transaction },
```

Comment:
This is possible because of the [undocumented `transaction` option param](https://github.com/sequelize/sequelize/blob/v4/lib/transaction.js#L38).